sw/device/lib/testing/aon_timer_testutils.c

```diff
@@ -22,7 +22,7 @@ void aon_timer_testutils_wakeup_config(dif_aon_timer_t *aon_timer,
   CHECK_DIF_OK(dif_aon_timer_irq_acknowledge(aon_timer,
                                              kDifAonTimerIrqWkupTimerExpired));
 
-  bool is_pending;
+  bool is_pending = true;
```

Comment:
Seems like an unrelated change, was the CI tripping?

Reply:
That was a request from @mundaym when I was using a copy of this funciton in the same `.c`. Initialize the flag to a value that will fail in the check in case the function below doesn't write anything in the variable.

Reply:
This is okay, but prefer making this changes either a separate commit or a separate PR.

Reply:
I splitted it in different commits.